tests/backup/backup_rancher_test.go

```diff
@@ -474,3 +479,323 @@ var _ = Describe("{NamespaceMoveFromProjectToProjectToNoProjectWhileRestore}", f
 		CleanupCloudSettingsAndClusters(backupLocationMap, credName, credUid, ctx)
 	})
 })
+
+// This testcase takes backup and restore of multiple namespaces belonging to multiple projects
+var _ = Describe("{MultipleProjectMultipleNamespacesBackupAndRestore}", func() {
```

Comment:
Can we change the test case name to `MultipleProjectsAndNamespacesBackupAndRestore` ?

Reply:
done